acp/dac/source_hub_js.go

```diff
@@ -44,25 +170,90 @@ func (a *SourceHubDocumentACP) AddPolicy(
 	policyMarshalType acpTypes.PolicyMarshalType,
 	creationTime *protoTypes.Timestamp,
 ) (string, error) {
-	panic("warning js build does not support sourcehub")
+	results, err := callJSFunction("acp_AddPolicy", policy, int(policyMarshalType))
```

Comment:
todo: Please add documentation, maybe at the top of the file, that gives readers context around what these JS functions pertain to (what JS package do they come from, etc).